torch_geometric/data/hetero_data.py

```
@@ -891,3 +897,17 @@ def to_homogeneous_edge_index(
         edge_index = torch.cat(edge_indices, dim=-1)
 
     return edge_index, node_slices, edge_slices
+
+
+def to_homogeneous_label(data: HeteroData, ) -> Tensor:
+    labeled_node_types = []
+    y = []
+    for i, node_type in enumerate(data.node_types):
+        if 'y' in data[node_type].keys():
+            labeled_node_types.append(i)
+            y.append(data[node_type].y)
+        else:
+            y.append(-1 * torch.ones(data[node_type].num_nodes))
+
+    if labeled_node_types:
```

when would this happen? not sure if failing silently like this makes sense